### PR TITLE
[INTERNAL] README.md: Migrate yarn setup to npm

### PR DESCRIPTION
**Pull request by flovogt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,45 +84,45 @@ The following needs to be done just once per setup.
...
 	In the application directory, execute:
     ```sh
     ui5 remove sap.f sap.m sap.ui.core themelib_sap_fiori_3
     ```
     ```sh
-    yarn add @openui5/sap.f
-    yarn add @openui5/sap.m
-    yarn add @openui5/sap.ui.core
-    yarn add @openui5/themelib_sap_fiori_3
+    npm add @openui5/sap.f
```
Comment (matz3) [inline README.md:100]: `npm install`

File: README.md [hunk 2]
```diff
@@ -91,45 +84,45 @@ The following needs to be done just once per setup.
...
 To return to using the OpenUI5 npm packages
 
-1. Remove the dependencies via Yarn
+1. Remove the dependencies via npm
     ```sh
-    yarn remove @openui5/sap.f
-    yarn remove @openui5/sap.m
-    yarn remove @openui5/sap.ui.core
-    yarn remove @openui5/themelib_sap_fiori_3
+    npm remove @openui5/sap.f
```
Comment (matz3) [inline README.md:122]: `npm uninstall`

Comment (flovogt) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (flovogt) [reply README.md]: Done

**Review by matz3 — DISMISSED** [review]

**Review by ecker — APPROVED** [review]
LGTM

**Pull request merged**